src/clr-angular/data/_variables.tables.scss

```diff
@@ -1,34 +1,43 @@
-// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
 // This software is released under MIT license.
 // The full license information can be found in LICENSE in the root directory of this project.
 
 // Theme variables
 // Usage: ./data/_tables.clarity.scss
 // Usage: ./data/datagrid/_datagrid.clarity.scss
-$clr-table-bgcolor: $clr-white !default;
-$clr-thead-bgcolor: $clr-near-white !default;
+$clr-table-bgcolor: $clr-color-neutral-0 !default;
+$clr-thead-bgcolor: $clr-color-neutral-50 !default;
 
 // Usage: ./data/datagrid/_datagrid.clarity.scss
-$clr-table-header-border-bottom-color: $clr-light-midtone-gray;
-$clr-table-footer-border-top-color: $clr-light-midtone-gray;
-$clr-table-border-color: $clr-lighter-midtone-gray;
+$clr-table-header-border-bottom-color: $clr-color-neutral-400 !default;
+$clr-table-footer-border-top-color: $clr-color-neutral-400 !default;
+$clr-table-border-color: $clr-color-neutral-300 !default;
 
 // Usage: ./data/_tables.clarity.scss
-$clr-tablerow-bordercolor: $clr-light-gray !default;
-$clr-table-bordercolor: $clr-light-midtone-gray !default;
+$clr-tablerow-bordercolor: $clr-color-neutral-200 !default;
+$clr-table-bordercolor: $clr-color-neutral-400 !default; // deprecated <= figure out how to get rid of this duplicate var
+$clr-table-border-radius: $clr-global-borderradius !default;
 $clr-table-borderwidth: 1px !default;
 $clr-table-smallpadding: 0.25rem !default;
+
 // Border radius of corner cells needs to be slightly less than the table's,
 // To make sure they correctly cover the whole area up to the border.
-$clr-table-cornercellradius: calc(#{$clr-default-borderradius} - 1px) !default;
+$clr-table-cornercellradius: calc(#{$clr-table-border-radius} - 1px) !default;
 
 // Usage: ./data/_tables.clarity.scss
 // Usage: ./data/datagrid/_datagrid.clarity.scss
 $clr-table-borderstyle: $clr-table-borderwidth solid $clr-table-bordercolor !default;
-$clr-table-font-color: clr-getTextColor($clr-table-bgcolor, text);
+$clr-table-font-color: $clr-color-neutral-700 !default;
+
 // Table dims and sizing
 $clr-table-fontsize: 0.541667rem;
 $clr-table-lineheight: 0.5833334rem !default;
+$clr-table-lineheightPaddingShim: (1rem - $clr-table-lineheight)/2 !default;
 $clr-table-cellpadding: 0.5rem !default;
 $clr-table-topcellpadding: $clr-table-cellpadding - $clr-rem-1px !default;
 $clr-table-bottomcellpadding: $clr-table-cellpadding - $clr-rem-1px !default;
+$clr-table-topcellpadding-cmp: $clr-table-cellpadding/2 - $clr-rem-1px !default;
+$clr-table-bottomcellpadding-cmp: $clr-table-cellpadding/2 - $clr-rem-1px !default;
+
+$clr-table--compact-rowHeight: 1rem !default;
```

Comment:
it's carried forward from a previous BEM style we used in 2016. the `--` went in front of the modifier. it might make sense to deprecate that and go to our kebab case names when we do #2162